RELEASING.adoc

```diff
@@ -51,6 +73,9 @@ All uploaded javadocs will be published under the following URL:
 https://googleapis.dev/java/spring-cloud-gcp/{BRANCH_VERSION_NAME}/index.html
 ----
 +
-Example: If you published the javadocs for version `1.2.2.BUILD-SNAPSHOT`, then the URL would be: https://googleapis.dev/java/spring-cloud-gcp/1.2.2.BUILD-SNAPSHOT/index.html
+Example: If you published the javadocs for version `2.0.0-SNAPSHOT`, then the URL would be: https://googleapis.dev/java/spring-cloud-gcp/2.0.0-SNAPSHOT/index.html
 
 5. For GA releases, ask on the https://gitter.im/spring-cloud-gcp/Lobby[Spring Cloud GCP Gitter] for the Spring team to update the https://spring.io/projects/spring-cloud-gcp#learn[API Documentation link] to point to the published Javadocs of the correct version.
```

Comment:
What do we need to do to get the reference documentation published and also pointed to on the project web site?

Reply:
I don't know - we talked about using our own GitHub pages to do this and having their pages link to us, rather than having them host our docs.

Reply:
I guess let's track this in a separate issue then.

Reply:
https://github.com/GoogleCloudPlatform/spring-cloud-gcp/issues/141